src/solvers/dgsem_tree/dg_2d_compressible_euler.jl

```diff
@@ -264,8 +264,8 @@ end
         u_prim[i, j, 2] = v1
         u_prim[i, j, 3] = v2
         u_prim[i, j, 4] = p
-        u_prim[i, j, 5] = log(rho)
-        u_prim[i, j, 6] = log(p)
+        u_prim[i, j, 5] = log_(rho)
+        u_prim[i, j, 6] = log_(p)
```

Comment:
Please don't change explicit SIMD stuff like this since it will make the optimizations impossible.